apps/src/applab/applab.js

```diff
@@ -292,14 +307,70 @@ function adjustAppSizeStyles(container) {
 var drawDiv = function () {
   var divApplab = document.getElementById('divApplab');
   divApplab.style.width = Applab.appWidth + "px";
-  divApplab.style.height = Applab.appHeight + "px";
+  divApplab.style.height = Applab.footerlessAppHeight + "px";
   if (Applab.levelHtml === '') {
     // On clear gives us a fresh start, including our default screen.
     designMode.loadDefaultScreen();
     designMode.serializeToLevelHtml();
   }
+
+  if (studioApp.share) {
+    renderFooterInSharedGame();
+  }
 };
 
+function renderFooterInSharedGame() {
+  var divApplab = document.getElementById('divApplab');
+  var footerDiv = document.createElement('div');
+  footerDiv.setAttribute('id', 'footerDiv');
+  divApplab.parentNode.insertBefore(footerDiv, divApplab.nextSibling);
```

Comment:
Even if the footer is outside of the divApplab, I believe we'll want divApplab to be 446 (so that when we remove other superfluous buttons a 480 pixel screen consists of the divApplab with the small footer on the bottom.